documentation/modules/auxiliary/scanner/http/robots_txt.md

```diff
@@ -0,0 +1,51 @@
+## Description
+
+This module will detect robots.txt files on web servers and analize its content.
+This type of file can reveal interesting information about areas of the site that are not indexed.
+
+## Vulnerable Application
+
+This scanner work perfectly with one of the [Google](https://www.google.com/) servers.
```

Comment:
While this isn't really a 'hack' lets not encourage targeting google.  I'd change this section out to mirror the other md 'You can use almost any web application to test this module, as robots.txt is extremely common.'